mkdocs/docs/how-to-release.md

```diff
@@ -130,6 +130,15 @@ Run the [`Python release` Github Action](https://github.com/apache/iceberg-pytho
 * Tag: Use the newly created tag.
 * Version: Set the `version` to `main`, as the source cannot be modified.
 
+Or if `gh` is available, trigger the action using:
+
+```bash
+gh workflow run python-release.yml --repo apache/iceberg-python --ref ${GIT_TAG} --field version=main
```

Comment:
How about instead of having to trigger this manually, we could also trigger this by the tag  creation. That's one of the first steps of the RC process: https://py.iceberg.apache.org/how-to-release/#create-tag This would trigger a job without having to do anything manual, similar to Iceberg-Go: https://github.com/apache/iceberg-go/blob/c1ffe04d56e31e475b52ce75fcbf16818a6e1169/.github/workflows/rc.yml#L24-L25